cmd/thv/app/proxy_stdio.go

```diff
@@ -0,0 +1,57 @@
+package app
+
+import (
+	"fmt"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stacklok/toolhive/pkg/logger"
+	"github.com/stacklok/toolhive/pkg/transport"
+	"github.com/stacklok/toolhive/pkg/workloads"
+)
+
+var proxyStdioCmd = &cobra.Command{
+	Use:   "stdio WORKLOAD-NAME SERVER_NAME",
+	Short: "Create a stdio-based proxy for an MCP server",
+	Long: `Create a stdio-based proxy that connects stdin/stdout to a target MCP server.
+
+Example:
+  thv proxy stdio my-workload my-server-proxy
+`,
+	Args: cobra.ExactArgs(2),
+	RunE: proxyStdioCmdFunc,
+}
+
+func proxyStdioCmdFunc(cmd *cobra.Command, args []string) error {
+	ctx, cancel := signal.NotifyContext(cmd.Context(), syscall.SIGINT, syscall.SIGTERM)
+	defer cancel()
+
+	workloadName := args[0]
+	serverName := args[1]
```

Comment:
nit: Is server name needed for anything other than logging? Why not just use workloadname?